ast/src/analyzed/mod.rs

```diff
@@ -711,6 +711,7 @@ impl SolvedTraitImpls {
     /// This just updates the `index` fields.
     /// Assumes that `to_remove` is sorted.
     pub fn remove_trait_impls(&mut self, to_remove: &[usize]) {
+        #[allow(clippy::iter_over_hash_type)]
```

Comment:
this is deterministic also because of no shared state?

Reply:
agreed, added comment

Reply:
actually, replaced by a BTreeMap